### DISCUSSION TITLE
Darwinia 2.0 Will Upgrade Staking Module, Which Allows Users To Earn Staking Rewards Via Nominating Collator

### DISCUSSION THREAD

**Discussion by 1022ranji** [darwinia-network/darwinia#960] [General]
Dear community,

To make the Darwinia protocol more concise and understandable, the Darwinia Dev Team has decided to study the feasibility of merging Darwinia Chain and Darwinia Parachain, which is called Darwinia 2.0. The latest research and testing progress of Darwinia 2.0 will be kept up to date with the community.

Darwinia 2.0 will consider updating the staking module, which allows users to earn staking rewards via nominating collator.

**Why consider updating the staking module?**

Once merged, Darwinia 2.0 will benefit from the economic security provided by the Polkadot Relay Chain validators. So Darwinia validators are no longer needed to ensure Darwinia’s economic security. But Darwinia 2.0 still needs [collators](https://wiki.polkadot.network/docs/learn-collator) to collect Darwinia’s transactions and produce state transition proofs for Relay Chain validators.

The responsibility of the collators is relatively small but also important. Users can still earn rewards by nominating collators. The reward ratio has not yet been determined, and the team will collect opinions from the community.

**Comment by SasoLithops** [comment]
Hi team!

I won't go into technical disscussion as you guys know how to and I don't 🙃
But as a nominator I am wondering how would that affect us?This staking model with locking Ring and minting Kton stays the same as you said in TG channel right? So nothing will change from nominator perspective?
But what will affect the ammount of Ring you would get. Will this stay the same in percentage value?
Since number of collators will be smaller and since collator does not have as huge role as validator does,does that mean that their rewards ammount should reduce compared to what validators charge now.They charge a lot on Crab chain. 
And the rest of rewards will they go to treasury?
I would support that idea.But would like to see more open public discussions about treasury funds and proposals when and why to withdraw them.This I think is the most important part of being a DAO.
Thank you for letting us participate in discussion🙌

**Reply by wuminzhe** [reply]
You are right. 
1. You can lock Ring and minting Kton. This will not be changed.
2. Compared with validators, the reward percentage of collators will decreases. But the ratio has not decided yet. 
3. But maybe because the number of collators decreases, the final income maybe will not drop drastically.

**Reply by SasoLithops** [reply]
Thank you Aki.

Will all options like automaticly increase ammount at staking,bond more,lock extra and others stay or is this being disscussed to change too?

This are some of my findings I had on Moonbeam which has this collators staking.These are some more questions I have:

Just saw Moonbeam implemented auto-compounding for staking where you can choose which ammount in % can be restaked.However due to our dual token system that probably not an option.

Also when I was staking at Moonbeam in the begining my collators often feel out of the active staking set.Have we decided on active number of collators?

Will we have the option to redelegate? I miss that on Moonbeam.You have to unbond there for one week, withdraw funds and then redelegate. I found this to be not attractive as you lose rewards during this week,but when your collators fall out of active set you have to do this.

**Reply by wuminzhe** [reply]
The team is discussing the details, there will be results soon.

**Reply by SasoLithops** [reply]
Noted 👍

**Reply by 2xJDubs** [reply]
Saso, this is a very good point.  How would one unbond with locked RING?  There will need to be an option to move Locked RING without unbonding.

**Reply by SasoLithops** [reply]
Agree. As I locked different ammounts at least 100 times for three years and would be best to just move if possible🤔

**Comment by 1022ranji** [comment]
Darwinia 2.0 related reading(will continue to update):
- [Darwinia 2.0] [Technical Feasibility Research And Test for Merging Solo Chain and Parachain](https://bit.ly/3UDOLgD)
- [Darwinia 2.0] [Darwinia 2.0 Merge Overview](https://medium.com/darwinianetwork/darwinia-2-0-merge-overview-96af96d668aa)
- [Darwinia 2.0] [Darwinia 2.0 Will Upgrade Staking Module, Which Allows Users To Earn Staking Rewards Via Nominating Collator](http://bit.ly/3hRR5Cf)
- [Darwinia 2.0] [Darwinia 2.0 Will Unify Address Format And Precision](http://bit.ly/3Fdpbd8)
- [Darwinia 2.0] [Darwinia 2.0 Staking Mechanism Adjustments](https://bit.ly/3FMYo7V)
- [Darwinia 2.0] [FAQs For Darwinia 2.0 Staking Mechanism Adjustments](https://bit.ly/3hTuzcO)
- [Darwinia 1.0] [How To Understand The Three Types Of Chains In The Darwinia Network](https://www.notion.so/1509a2d46d734bd993514e3e7298ace3)

**Comment by pmensik** [comment]
Hello, as a collator (Polkadotters), I have a few questions

 - what will be the initial bond?
 - how many collators is the network going to support?
 - is there any ETA for transition?

Thanks for a lot for the answers

**Reply by wuminzhe** [reply]
Sorry for being late.
* RING will be the initial bond for a collator.
* This has not been determined. 
* After Darwinia 2.0, all validators will go offline automatically. Then, anyone can apply to a collator. More details will be public soon.

**Comment by 2xJDubs** [comment]
In the most recent article on the new staking mechanism a change to block rewards is noted with 80% of block rewards now going to the treasury versus 50% in Darwinia 1.0. 

![image](https://user-images.githubusercontent.com/35436605/207228973-a75f0176-0b65-4f8c-be8b-8e7dcd896930.png)

This change seems extremely unfair to collators an delegators and doesn't make a lot of sense.  Currently the Darwinia Treasury holds 93.8M RING and over my time staking, validating, being an Ambassador and Council member for Darwinia I have only seen a small number of proposals and tips paid out.  So I am not sure why a hike in what the treasury receives is needed when little is done with the treasury. 

At this time I would vote no against the merge using this rewards split unless the team can provide a breakdown of how this balance compared to the current system  From what I currently understand of the new model this would significantly reduce rewards for delegators, especially long time supporters of the project that hold significant staking power in the current model.

I look forward to the team providing a clear picture of what 80% of block reward needs to be sent to the treasury.

**Reply by 2xJDubs** [reply]
To add to my comment about the amount of block rewards going to the treasury.   MoonBeam does not take 80% or even 50% of block rewards to their treasury.  They take 30%.  The rest is distributed to collators and delegators.  Now I don't think Darwinia needs to go with 30% as I feel the previous 50% was acceptable.

![image](https://user-images.githubusercontent.com/35436605/207231462-586f91d6-f030-4a35-b1f6-a51f0eecde1e.png)

**Comment by SasoLithops** [comment]
Hi again!
I agree with Jesse. Also looking forward for team to explain how it got to number 20% and how this 20% would be split among collators and nominators?
Also would like to know why can we only nominate to one collator?

**Reply by 1022ranji** [reply]
please check this
> * why 20%?
>   Compared with the validators, the responsibility and importance of the collators are much lower. As long as there is one collator working, the system can run normally. But 20 or 30 or other numbers, this has not been determined. But it will be lower than 50%. This will finally be decided through the on-chain governance of Darwinia 1.0
> * how this 20% would be split among collators and delegators(nominators)?
>   All collators share this 20% equally, and then each collator takes away the commission fee according to the commission rate set by himself, and the rest is shared equally with his supporters, that is, delegators(nominators).
> * why one collator?
>   In order to simplify the staking system and reduce the possibility of bugs.
>   Will improve in the future.
> * Why should Treasury take away 80%?
>   Treasury is the financial source of all governance. Treasury is also the financial source for parachain slots in the future.
>   In addition, a high proportion of Treasury can reduce liquidity. Increasing the proportion of Treasury and using it carefully is equivalent to a means of reducing the inflation rate.

**Comment by wuminzhe** [comment]
* why 20%?
  Compared with the validators, the responsibility and importance of the collators are much lower. As long as there is one collator working, the system can run normally. But 20 or 30 or other numbers, this has not been determined. But it will be lower than 50%. This will finally be decided through the on-chain governance of Darwinia 1.0
  
* how this 20% would be split among collators and delegators(nominators)?
  All collators share this 20% equally, and then each collator takes away the commission fee according to the commission rate set by himself, and the rest is shared equally with his supporters, that is, delegators(nominators).
  
* why one collator?
  In order to simplify the staking system and reduce the possibility of bugs. If you want to choose more than one collator, you can do it through multi accounts.
  Will improve in the future. 
  
* Why should Treasury take away 80%?
  Treasury is the financial source of all governance. Treasury is also the financial source for parachain slots in the future.
  In addition, a high proportion of Treasury can reduce liquidity. Increasing the proportion of Treasury and using it carefully is equivalent to a means of reducing the inflation rate.

**Comment by SasoLithops** [comment]
Thank you for reply.

1. I feel like there should be some basic approx calculations for average user to get a feeling what kind of rewards to expect given the percentage (for example for 20%,30%,40%) so that they do not all just vote on the highest percentage. However I feel this would be kind of hard since collator has own right to change comission.

3. Which brings me to choosing only one collator. This is not the best solution in my opinion. I feel like too much power is given to collator. He can daily change commision so as a nominator you have to daily check(many do not have time to do it I know from questions asked in TG chat, their validators changed commision to 90% but they had no idea and lost many rewards) Also if you no longer want to use this collator,can you easily redelegate? I know in Moonbeam you need to unstake,wait one week without rewards and then redelegate.Which is also not the best solution imho. How to protect nominators from bad actor collators?

4. As for last answer I do agree treasury is very important for each project. But not on the account of nominator getting huge drop in staking rewards. This would drive people away from ecosystem and even lower the usecase Kton has.

I do wish more of us could discuss this in public chat as many don't have Github.

**Reply by 1022ranji** [reply]
🤔 I think the efficiency and effectiveness of information delivery are diminishing. Information received on multiple social platforms is passed from the relevant person A to the corresponding developer B or C or D, and then the corresponding developer makes an answer to A, and then A reaches the various communities, so obviously the delivery time and the effectiveness of the information are discounted. However, there are some questions that must be answered by the corresponding technical person because the design of the code/mechanism is involved. In this case it is more appropriate to give preference to the tools commonly used by developers as a platform for discussion. GitHub, like Discord/Tg, requires only a simple registration to use. Also, the GitHub Discussion will contain all the historical questions, and will be a one-stop-show/answer for questions about Darwinia 2.0, which will also be user-friendly for users to understand all of the issues and updates.

PS: The information has been synced to the community, but we haven't received a lot of discussions except for the ambassador's feedback(maybe because of the market environment, the number of users who pay close attention to the progress/updates may still be relatively small.).

**Reply by ZolaxZola** [reply]
Hey @SasoLithops, to reply to point 2, in the Darwinia 2.0 Staking Mechanism you would be able to redelegate( change the collator you delegated) easily, without waiting 14 days.

**Reply by SasoLithops** [reply]
Hey Ran Ji!

I agree it is mostly developer related.  But this will majorly affect us all regular users too. And I think this should somehow be brought to their attention too, with really simplified explanation what this will do. Like you would be explaining to grandma😅 Most of Ring holders are not technical. I am sure many have no idea what Darwinia 2.0 will be/do. The ones that understand some ask will we have to do anything after merge. And that is it. My opinion is it is too technical for regular user.
I am sorry if my opinions feel to you devs like I am too critical,but I am doing this only from my love to the project.

**Reply by SasoLithops** [reply]
Hey Zola!

That is really good news👍 Makes much more sense chosing one collator as you can switch if some prove to be bad actors. 

Will we have an option to check what fees they charge on staking app?

**Reply by ZolaxZola** [reply]
> Hey Zola!
> 
> That is really good news👍 Makes much more sense chosing one collator as you can switch if some prove to be bad actors.
> 
> Will we have an option to check what fees they charge on staking app?

If you mean the commission ( The percent a collator takes off the top of the due staking rewards) 
---yes, this would be displayed

**Reply by SasoLithops** [reply]
Yes sorry ,I meant  the commision.Great👍
